test/clj/election/controllers/elections_test.clj

```diff
@@ -0,0 +1,52 @@
+(ns election.controllers.elections-test
+  (:use clojure.test)
+  (:require
+    [election.controllers.elections :as e]))
+
+(deftest email-validation
+  (testing "accepts gmail email"
+    (let [result (e/is-valid-email? "someone@gmail.com")]
+      (is result)
+    )
+  )
+  (testing "accepts email with dots"
+    (let [result (e/is-valid-email? "a.b.c@yahoo.com")]
+      (is result)
+    )
+  )
+  (testing "accepts special gmail with +"
```

Comment:
Mind adding a couple more tests for other than `.com` ending domains?
Like `agilealliancebrazil.org.br`

Reply:
Sure. I will do so.